java/jdbc/ConnectionSamples/JDBCDBTokenSample.java

```diff
@@ -0,0 +1,98 @@
+
+/* Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.*/
+/*
+   DESCRIPTION    
+   The code sample shows how to use the JDBC driver to establish a connection
+   to the Autonomous Database (ADB) using database token 
+   issued by the OCI Identity service. 
+
+   You need to use either JDBC driver to use
```

Comment:
I don't know what "either" is referring to. Are there two JDBC drivers that can be used? Also the "n" is missing from "authenticatio"